draft-ietf-tls-esni.md

```diff
@@ -934,6 +937,17 @@ IANA is requested to create an entry, echo_required(121) in the
 existing registry for Alerts (defined in {{!RFC8446}}), with the
 "DTLS-OK" column being set to "Y".
 
+# ECHOConfig Extension Guidance {#config-extensions}
+
+Any future information or hints that influence the outer ClientHello SHOULD be
+specified as ECHOConfig extensions. This is primarily because the outer ClientHello
```

Comment:
Or, I guess, as ECHOConfigV2, right?

Reply:
Yeah, I'll add that: "as ECHOConfig extensions, or an entirely new version of ECHOConfig."